chirp/drivers/baofeng_uv17Pro.py

```diff
@@ -1066,3 +1069,11 @@ class UV17ProGPS(UV17Pro):
     _has_skey2_short = True
     VALID_BANDS = [UV17Pro._airband, UV17Pro._vhf_range, UV17Pro._vhf2_range,
                    UV17Pro._uhf_range, UV17Pro._uhf2_range]
+
+
+@directory.register
+class BF5RM(UV17Pro):
+    VENDOR = "Baofeng"
+    MODEL = "5RM"
+    VALID_BANDS = [UV17Pro._airband, UV17Pro._vhf_range, UV17Pro._vhf2_range,
+                   UV17Pro._uhf_range, UV17Pro._uhf2_range]
```

Comment:
Is this actually different from the thing it inherits from? The `VALID_BANDS` look identical to L1070 above. If not, it might be better to just add this as a website alias (see all the "use..." notes on the front page)

Reply:
Yes, airband is added for this radio.